### PR TITLE
Add list, completion command skeleton

### PR DESCRIPTION
**Pull request by asbubam**
# `list` command
* listing locust deployments that starts with prefix (locust-master-)
* example
```bash
 # If locust deployments exists.
$ ./build/klocust list
>>> 1 locust deployments in klocust namespace. (PREFIX: locust-master-)
+---------------------+-------+-------+------------+-----------+---------+
|          DEPLOYMENT |  NAME | READY | UP-TO-DATE | AVAILABLE |     AGE |
+---------------------+-------+-------+------------+-----------+---------+
| locust-master-hello | hello |   0/1 |          1 |         0 | 4h7m23s |
+---------------------+-------+-------+------------+-----------+---------+

 # If locust deployments does not exists.
$ ./build/klocust list -n wrong
>>> 0 locust deployments in wrong namespace. (PREFIX: locust-master-)
```

# `completion` command
* Outputs shell completion for the given shell (bash or zsh

### CODE REVIEW

**Review by jupitersong — APPROVED** [review]
👍

File: internal/klocust/klocust.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+	fmt.Printf("%d locust deployments in %s namespace. (PREFIX: %s)\n",
+		len(locustDeployments), namespace, LocustMasterDeploymentPrefix)
+
+	if len(locustDeployments) <= 0 {
+		return nil
+	}
+
+	writer := tabwriter.NewWriter(os.Stdout, 0, 8, 1, '\t', tabwriter.AlignRight)
+	if _, err := fmt.Fprintf(writer, "\nDEPLOYMENT\tNAME\tREADY\tUP-TO-DATE\tAVAIABLE\tAGE\n"); err != nil {
```
Comment (GwonsooLee) [inline internal/klocust/klocust.go:53]: TableWriter 헤더는 아래와 같이 조금 보기 좋게 정리하실 수 있습니다
https://github.com/DevopsArtFactory/goployer/blob/2ded6e395ca844b68a4191b562361ee2dc723aa8/pkg/deployer/deployer.go#L415-L416

**Comment by asbubam** [comment]
* add `completion` command for command autocomplete
* move every AddCommand() function call to rootCmd.init()
ref: https://github.com/DevopsArtFactory/klocust/pull/2/commits/1cb69506c12a63da3a6de5148b6da5895e0e777f

File: internal/klocust/klocust.go (hunk 1 above)
Comment (asbubam) [reply internal/klocust/klocust.go]: Thanks!! I've changed to tablewriter for printing.

**Review by GwonsooLee — APPROVED** [review]
👍  감사합니다~ :)

**Pull request merged**